.gitignore

```diff
@@ -2,3 +2,6 @@
 *.jpeg
```

Comment:
Prolly wanna add *.jpg here too

Reply:
I think we can get rid of that .gitignore since theres another one in the main folder (that one has jpg)